ripple/src/main/test/script.test.js

```diff
@@ -1,6 +1,10 @@
-/* Contains unit tests for script.js file */
+/* Contains unit tests for script.js file. Currently has ~50% coverage for all script functions (doesn't test functions that just call 
```

Comment:
not blocking but does it spit out a graph or something that shows this? Just asking because that is something that would be cool to showcase in your presentation or some document that can be referenced in the final eval.

Reply:
Will look into that! I just manually calculated it for this.